src/main/java/org/apache/sling/serviceusermapping/impl/ServiceUserMapperImpl.java

```diff
@@ -487,18 +487,24 @@ private boolean isValidUser(final String userId, final String serviceName, final
             return false;
         }
         List<ServiceUserValidator> validators = getUserValidators();
-        if (!validators.isEmpty()) {
+        if (validators.isEmpty()) {
+            if (require) {
+                log.debug("isValidUser: No active validators for userId [{}] and require -> invalid", userId);
+                return false;
+            } else {
+                log.debug("isValidUser: No active validators for userId [{}] -> valid", userId);
```

Comment:
same here:
`"isValidUser: No active validators for userId '{}' -> valid", userId`
instead of [] around the id.

Reply:
done